pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -148,7 +149,9 @@ func (pc *PackageControllerClient) GetCuratedPackagesRegistries() (sourceRegistr
 			defaultImageRegistry = gatedOCINamespace
 		}
 	} else {
-		defaultImageRegistry = strings.ReplaceAll(defaultImageRegistry, eksaDefaultRegion, pc.eksaRegion)
+		if pc.eksaRegion != eksaDefaultRegion {
+			defaultImageRegistry = strings.ReplaceAll(defaultImageRegistry, eksaDefaultRegion, pc.eksaRegion)
```

Comment:
nit: This is really not necessary. Even if the check wasn't done, this will still work.

Reply:
the same

Reply:
Agreed, but I thought it better captured the intention.